### ISSUE TITLE
Refuses to install

### ISSUE THREAD

**Issue by baudneo** [openrazer/openrazer#1639]
### Distribution

Xubuntu 20.04

### Support

- [x] Have you tried rebooting?
- [x] Have you looked at the [Troubleshooting](https://github.com/openrazer/openrazer/wiki/Troubleshooting) page?

Trying to install openrazr using the .deb pkg and it refuses to install. It is saying it cant install itself because it depends on itself.

```
❯ sudo dpkg --ignore-depends=openrazer-meta -i ~/Downloads/openrazer-meta_2.8.0-0_all.deb
(Reading database ... 400666 files and directories currently installed.)
Preparing to unpack .../openrazer-meta_2.8.0-0_all.deb ...
Unpacking openrazer-meta (2.8.0-0) over (2.8.0-0) ...
dpkg: dependency problems prevent configuration of openrazer-meta:
 openrazer-meta depends on openrazer-driver-dkms (= 2.8.0-0); however:
  Package openrazer-driver-dkms is not installed.
 openrazer-meta depends on openrazer-daemon (= 2.8.0-0); however:
  Package openrazer-daemon is not installed.
 openrazer-meta depends on python3-openrazer (= 2.8.0-0); however:
  Package python3-openrazer is not installed.

dpkg: error processing package openrazer-meta (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 openrazer-meta
```

**Comment by z3ntu** [comment]
You also need to install the openrazer-driver-dkms, openrazer-daemon and python3-openrazer debs as well, openrazer-meta is just a meta-package depending on those packages.

Also why do you want to install 2.8.0 from debs?

**Comment by baudneo** [comment]
Thank you I added the deb repos and installed those with --fix-broken install. Now though it says it cannot recognize the device? It was working a few days ago, I will also post the openrazer logs with screen shot. I reinstalled because I thought that might help but I didnt purge the packages, could that be it?

Gist -> [here](https://gist.github.com/baudneo/bc539e7cb39d12d2b1026d5b80874976)

![openrzr_issue](https://user-images.githubusercontent.com/86508179/137019547-892f6b90-57bb-467f-8e43-d6ede9922ca4.png)
